lib/filepicker_rails/filepicker_image_url.rb

```diff
@@ -0,0 +1,141 @@
+# Creates the full path of the image to the specified `url` accepts optional
+# `options` hash for configuration.
+#
+# #### Constructor
+#
+# @param [String] url                       Base filepicker url
+# @param [Hash] options
```

Comment:
I think you can fix this error with the same behavior described here https://github.com/Ink/filepicker-rails/pull/133/files#r40270205